.travis.yml

```diff
@@ -12,6 +12,9 @@ script:
   - go get -u github.com/mholt/caddy
   - go get -u gopkg.in/natefinch/lumberjack.v2
   - go get -u github.com/miekg/dns
+  - go get -u github.com/gomods/athens/...
+  - rm -rf ${TRAVIS_HOME}/gopath/src/github.com/gomods/athens/vendor/github.com/spf13/afero
```

Comment:
Because it tries to use the one from athens's vendor inside our project and we end up with an error like this:
```
./gomods.go:125:49: cannot use fs (type "github.com/spf13/afero".Fs) as type "github.com/gomods/athens/vendor/github.com/spf13/afero".Fs in argument to module.NewGoGetFetcher:
```

Reply:
Couldn't find anything or maybe used the wrong keywords for search. Handled it the ugly way for now till we find a better solution.